lib/mayaUsd/python/wrapPrimUpdaterManager.cpp

```diff
@@ -117,6 +117,11 @@ std::string readPullInformationString(const PXR_NS::UsdPrim& prim)
     return dagPathStr;
 }
 
+bool isEditedPrimOrphaned(const PXR_NS::UsdPrim& prim)
+{
+    return MAYAUSD_NS_DEF::isEditedAsMayaOrphaned(prim);
```

Comment:
Just use the namespace name "MayaUsd::"

Reply:
I was using the same as existing code above it. I will change both.